### ISSUE TITLE
Keystrokes, "anounce last tooltips" should be inserted to NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#866]
**Reported by RamboUdin on 2010-08-29 06:13**
When someone missed the tooltips, say for example, "updates are ready for your computer: click here to install the updates", NVDA will not repeat it for users. For JAWS, (just for a while), it will repeat it when you press insert+TAB. So, this keystrokes should be inserted to NVDA so that users can here it again.

thanks!

**Comment by leonardder** [comment]
Cc @JosephSL

**Comment by Adriani90** [comment]
Closing in favor of #6922 which contains more discussion, see the last comment in that issue.
